### PR TITLE
[Keyboard] Fix VID and PID for AnnePro2

### PR DESCRIPTION
**Pull request by zv0n**
Fix of the AnnePro2's VID and PID

## Description

I have changed AnnePro2's VID and PID to the original firmware's values, so it can be used with VIA (0xFEED is invalid VendorId).

I got the original values from AnnePro2-Tools website - https://openannepro.github.io/ap2_revisions/

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zv0n** [comment]
It seems the CI is failing on files that I have not touched, how should I proceed here?

**Comment by drashna** [comment]
> It seems the CI is failing on files that I have not touched, how should I proceed here?

That's fine.

**Review by drashna — APPROVED** [review]

File: keyboards/annepro2/info.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "url": "https://openannepro.github.io/",
     "maintainer": "bwisn",
     "usb": {
-        "vid": "0xFEED",
+        "vid": "0x04D9",
```
Comment (drashna) [inline keyboards/annepro2/info.json:6]: Technically, this looks to be holtek's VID. `0xAC20` would be better, I think.

Comment (zv0n) [reply keyboards/annepro2/info.json:6]: Yeah, looks like it, however it looks like Obins borrowed the VID, I even found it referenced in SDL - https://github.com/libsdl-org/SDL/blob/main/src/joystick/SDL_joystick.c

So if we want to keep the same VID/PID as the original keyboard, I think 0x04D9 is fine

However I'd understand if you want to change the VID to differentiate between the original keyboard and one with QMK

Comment (zv0n) [reply keyboards/annepro2/info.json]: @drashna I've changed the VID assuming we want to differentiate QMK from the original keyboard (if we don't want to differentiate, we should switch it back)

**Review by tzarc — APPROVED** [review]

**Pull request merged**